Parsers/ASimWebSession/Parsers/ASimWebSession.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Web Session ASIM parser
-  Version: '0.1'
-  LastUpdated: Nov 30, 2021
+  Version: '0.2'
```

Comment:
Even though nothing changed directly on this metaparser - you update the version? 
I think we need to set rules to how and when we promote version numbers

Reply:
Makes sense. This time I will keep it.